### PR TITLE
Update blurredLocation.js

### LINKED ISSUE #95: Add +/- zoom buttons to Leaflet map
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem

The map as it is now is hard to zoom around in on some interfaces (i.e. trackpad) , so let's add some zoom buttons, like on this Leaflet example:

http://leafletjs.com/examples/quick-start/

That example also says zoom controls should be on by default. Not sure why they aren't in this map! 

## Solution

Ah, so we have to remove the `{zoomControl: false}` part of this line:

https://github.com/publiclab/leaflet-blurred-location/blob/master/src/blurredLocation.js#L17

```js
options.map = options.map || new L.Map(options.mapID, { zoomControl: false }).setView([options.location.lat, options.location.lon], options.zoom);
```

I think we can just replace `{zoomControl: false}` with `{}`.

## Steps to Fix

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/wiki/developers) or go to [the chatroom](http://publiclab.org/chat) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

### PR DESCRIPTION
**Pull request by Bubblemelon**
This edit would fix #95

### CODE REVIEW

File: src/blurredLocation.js [hunk 1]
```diff
@@ -70,7 +71,7 @@ BlurredLocation = function BlurredLocation(options) {
   function geocodeStringAndPan(string, onComplete) {
     var url = "https://maps.googleapis.com/maps/api/geocode/json?address="+string.split(" ").join("+");
     var Blurred = $.ajax({
-        async: false,
+        async: options.map = options.map || new L.Map(options.mapID, {}).setView([options.location.lat, options.location.lon], options.zoom);,
```
Comment (jywarren) [inline src/blurredLocation.js]: Oh interesting! What does this line do?

Comment (Bubblemelon) [reply src/blurredLocation.js]: Hi @jywarren, thanks for pointing it out! Should be fixed now. :)

**Comment by jywarren** [comment]
Hi -- this looks perfect; merging now. If you're looking for a new challenge, please take a look at our `help-wanted` list on our main project! https://github.com/publiclab/plots2/labels/help-wanted

If you'd be able to try a fix for the markers issue on this repository, we're also really eager for some help with that one!

Thanks! 🎉

**Comment by Bubblemelon** [comment]
Hi @jywarren Thanks for the `help-wanted` link and for guiding me through. I'll be sure to help if I can. Also, just saw the open issue about the markers. I'll let a friend know who might be interested in helping. 😀

**Comment by jywarren** [comment]
awesome, thanks a million!

**Pull request merged**